io-ballerina/src/io/character/channel_string_io.bal

```diff
@@ -183,14 +183,14 @@ public function channelWriteLinesFromStream(WritableChannel writableChannel, str
     }
 }
 
-# Write a JSON to a channel.
+# Write a JSON to a channel
```

Comment:
How does this serialize the JSON when writing? What encoding does it use?

Reply:
Here all APIs are using default encoding as `UTF-8`. We are not concerned about other encoding types here.